package.json

```diff
@@ -23,18 +23,20 @@
     "async": "^1.3.0",
     "body-parser": "^1.13.1",
     "bower": "^1.4.1",
+    "cfenv": "~1.0.0",
     "chalk": "^1.1.0",
     "compression": "^1.5.0",
     "connect-flash": "~0.1.1",
     "connect-mongo": "~0.8.1",
     "consolidate": "~0.13.1",
     "cookie-parser": "^1.3.2",
+    "crypto": "0.0.3",
     "express": "^4.13.1",
```

Comment:
Is node's built-in crypto not good enough/have what we need? For something like this where the password could just be pick 5 letters from the alphabet, I don't really wanna add a library for this.